app/src/main/java/com/github/swent/echo/viewmodels/HomeScreenViewModel.kt

```diff
@@ -169,6 +175,8 @@ constructor(
     // End of methods to set the filters container values
 
     private fun filterEvents() {
+        print("section : $section\n")
+        print("semester : $semester\n")
```

Comment:
Good for testing, but shuold be removed from main.

Reply:
again, I debugged some stuff but forgot to remove it 🥲